Signal/src/Storyboard/Main.storyboard

```diff
@@ -912,9 +912,6 @@
                                         </label>
                                     </subviews>
                                 </tableViewCellContentView>
-                                <connections>
-                                    <segue destination="tWr-1v-Sq9" kind="push" id="23M-9X-9xV"/>
-                                </connections>
```

Comment:
I don't think you need these segue changes any more. Can you revert them and test that it still works?